### PR TITLE
Prettify the wi-fi reporting and add more description

### PR DESCRIPTION
**Pull request by drewvolz**
Part of #2054

pretty-wifi

### CODE REVIEW

File: source/views/help/wifi.js [hunk 1]
```diff
@@ -80,15 +81,45 @@ export class ReportWifiProblemView extends React.Component<Props, State> {
...
+        <Card footer={this.state.status}>
+          <Container>
+            <Title selectable={true}>Report a Wi-Fi Problem</Title>
+            <Description selectable={true}>
+              If you are having an issue connecting to any of the St. Olaf
+              College Wi-Fi networks, please tap the button below.
+            </Description>
+            <Description selectable={true}>
+              We will record your current location and what device you are
+              using, then send it to a server that IT maintains. They are
```
Comment (rye) [inline source/views/help/wifi.js]: >They are planning to look closely at where people are having issues!

We don't really speak on their behalf; I suppose I would be more neutral?  As in,

>The networking team can then use this information to identify where people are having issues!

Comment (rye) [reply source/views/help/wifi.js]: Also, we should be clear that this information is anonymous, and we do not collect usernames.

File: source/views/help/wifi.js [hunk 2]
```diff
@@ -80,15 +81,45 @@ export class ReportWifiProblemView extends React.Component<Props, State> {
...
+      <View>
+        <Card footer={this.state.status}>
+          <Container>
+            <Title selectable={true}>Report a Wi-Fi Problem</Title>
+            <Description selectable={true}>
+              If you are having an issue connecting to any of the St. Olaf
+              College Wi-Fi networks, please tap the button below.
+            </Description>
+            <Description selectable={true}>
+              We will record your current location and what device you are
```
Comment (rye) [inline source/views/help/wifi.js]: >and what device you are using

could be something more like

>and some general information about the device you are using

File: source/views/help/wifi.js (hunk 1 above)
Comment (rye) [reply source/views/help/wifi.js]: I think it's okay to be slightly more wordy here just to be clear about what we're suggesting.  Alternatively, we could add some of this to the legal section and point people that way.

File: source/views/help/wifi.js [hunk 3]
```diff
@@ -80,15 +81,45 @@ export class ReportWifiProblemView extends React.Component<Props, State> {
...
   render() {
     return (
-      <ScrollView>
-        <Card footer={this.state.status} header="Report a WiFi Problem!">
-          <Button
-            disabled={this.state.status !== ''}
-            onPress={this.start}
-            title="Report"
-          />
+      <View>
```
Comment (hawkrives) [inline source/views/help/wifi.js]: We don't need this View anymore

**Review by hawkrives — CHANGES_REQUESTED** [review]
Rather than use a Container inside the Card, I'd rather use the `style={}` prop of Card itself to add the padding.

File: source/views/help/wifi.js (hunk 2 above)
Comment (drewvolz) [reply source/views/help/wifi.js]: Should we change that in the release notes, too?

File: source/views/help/wifi.js (hunk 1 above)
Comment (drewvolz) [reply source/views/help/wifi.js]: Again, if we change this, let’s be sure the release notes read the same.

**Comment by rye** [comment]
I have a riff on this that I will PR into this PR.

**Comment by rye** [comment]
See #2063 for an idea.

**Comment by drewvolz** [comment]
I've removed the outside view, the inner container, applied styles to the card, and tweaked the wording to be generic yet transparent.

**Review by hawkrives — APPROVED** [review]

**Pull request merged**